### DISCUSSION TITLE
Plugging in GCS bucket data to a new mimir deployment

### DISCUSSION THREAD

**Discussion by luvpreetsingh** [grafana/mimir#8331] [Help and support]
Hi everyone,

I am using distributed mimir in kubernetes deployed using helm in microservices mode.
The helm chart version is `5.0.0`.

I have a mimir running with GCS bucket as a backend. Now, I want to shut down this mimir and just plugin the bucket to a new mimir I will launch in a new cluster.

What things do I need to take care of 

* while shutting down the current mimir
* while connecting the bucket to the new mimir
* OR will it work out of the box?

This is also in keeping view of any disaster recovery we might have to do with mimir in case that ever happens.

**Comment by luvpreetsingh** [comment]
If I have mimir chunks in a GCS bucket and I plug that bucket into a new mimir, will it be able to query the older data automatically?

**Reply by pracucci** [reply]
> If I have mimir chunks in a GCS bucket and I plug that bucket into a new mimir, will it be able to query the older data automatically?

Yes, assuming that:

- The two Mimir clusters will not run simultaneously (you shutdown the old first, and then you start up the new one)
- The tenant IDs don't change between the two Mimir clusters
